core/src/test/java/com/radixdlt/recovery/REv2LedgerRecoveryTest.java

```diff
@@ -123,7 +123,7 @@ private DeterministicTest createTest() {
                     StateComputerConfig.rev2(
                         Network.INTEGRATIONTESTNET.getId(),
                         TransactionBuilder.createGenesisWithNumValidators(
-                            1, UInt64.fromNonNegativeLong(10)),
+                            1, UInt64.fromNonNegativeLong(Long.MAX_VALUE)),
```

Comment:
There are still epoch changes in rev2 state computer, even though this test uses a "no epochs" node module, which messes up safety store recovery (`NoEpochsSafetyRecoveryModule` is stripped of the epochs logic that `EpochsSafetyRecoveryModule` has). Not sure if this is the cleanest solution, but seems reasonable(?)

Reply:
Haha good spot.

Hmm tbh I'm not sure if we even need `NoEpoch` tests?

Like it might be nice if we could just make no epoch = long epochs? Would certainly decrease the number of supported combinations of the system and make it less likely we overlook something.

What do you think @LukasGasior1 ? @talekhinezh ?